### PR TITLE
Change Documentation for Explicit Interface

### PR DESCRIPTION
**Pull request by MaximeBouton**
Remove duplicate with tutorials in [POMDPExamples.jl](https://github.com/JuliaPOMDP/POMDPExamples.jl)

### CODE REVIEW

File: docs/src/explicit.md [hunk 1]
```diff
@@ -1,203 +1,62 @@
...
-using QMDP
-@requirements_info QMDPSolver() TigerPOMDP() 
+transition(pomdp, s, a) 
+observation(pomdp, a, sp)
+initialstate_distribution(pomdp)
+isterminal(pomdp)
 ```
 
-We will begin by implementing the state, action, and observation spaces and functions for initializing them and sampling from them.
+The functions `transition`, `observation`, and `initialstate_distribution` return a distribution over state. That is it possible to sample a state from this distribution as well as computing the pdf of a state given this distribution. `transition` represents the probability distribution of the next state given a current state s and action a. `observation` represents the probability distribution of an observation given the true underlying state sp and action a.
```
Comment (Shushman) [inline docs/src/explicit.md]: That is it possible -> This makes it possible

**Comment by Shushman** [comment]
Made one little typo suggestion but otherwise looks good. We'll need a similar revamp for generative too right?

**Comment by zsunberg** [comment]
It's already done for generative

**Comment by zsunberg** [comment]
(there is a broken in generative link but I'm fixing it now)

**Pull request merged**